### PR TITLE
Route 13 Date voucher issued translation

### PR DESCRIPTION
**Pull request by dtgreiner**
## _Merging this PR_
- use the squash-merge strategy for PRs targeting a release-X branch
- use a merge-commit or rebase strategy for PRs targeting the stable branch

## Description
[//]: # (Summarize changes and include links related issue. List any new dependencies)
Add a translation for the current 'Date voucher issued' in Route 13 Accept Referral Step

## Type of change
[//]: # (e.g., Bug fix, New feature, Code clean-up, Dependency update)
New feature

## Checklist before requesting review
[//]: # (Remove any items that are not applicable)
- [X] I have performed a self-review of my code
- [X] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [ ] I have updated the documentation (or not applicable)
- [ ] I have added spec tests (or not applicable)
- [X] I have provided testing instructions in this PR or the related issue (or not applicable)

### CODE REVIEW

**Comment by dtgreiner** [comment]
Screenshot 2024-12-03 at 3 03 41 PM

File: app/views/match_decisions/thirteen/_accept_referral.haml [hunk 1]
```diff
@@ -8,7 +8,7 @@
               .c-choice-icon.c-choice-icon--confirm.mr-4
               %h3 Accept Match
           .o-choose__content
-            = form.input :date_voucher_issued, label: 'Date voucher issued', as: :date_picker, disabled: !@decision.editable?
+            = form.input :date_voucher_issued, label: Translation.translate('Route Thirteen Date voucher issued'), as: :date_picker, disabled: !@decision.editable?
```
Comment (eanders) [inline app/views/match_decisions/thirteen/_accept_referral.haml]: Hmm, if we're going to add `Route Thirteen` we should probably provide a default translation to the previous version.  What do you think about just translating `Date voucher issued`?

Comment (dtgreiner) [reply app/views/match_decisions/thirteen/_accept_referral.haml]: That makes sense to me, I almost went that way originally and it should be a simple update. I'll get it pushed to the PR here soon.

Comment (dtgreiner) [reply app/views/match_decisions/thirteen/_accept_referral.haml]: This has been added in [this commit](https://github.com/greenriver/boston-cas/pull/886/commits/4ede0bbd22c83748f54321c4e4b22632d3a6373c)

**Review by eanders — APPROVED** [review]

**Pull request merged**